### ISSUE TITLE
Contractor - Review Documents

### ISSUE THREAD

**Issue by jackcarlisle** [karmaradio/karma#10]
As a contractor,
I want to be able to review project documents,
So that I can decide if I want to join that project

**Comment by JMurphyWeb** [comment]
links to documents can appear in their offer on the dashboard. linked to #19

**Comment by jonscottfilm** [comment]
some of these documents will be FYI purposes only (eg. [General Safety Practices](https://drive.google.com/file/d/0B6h4FGwRepkSSUVIeXlMOWk3X3c/view?usp=sharing))
some of these documents will be essential to say you've seen them (eg. [Prohibition of Harrassment Policy](https://drive.google.com/file/d/0B6h4FGwRepkSV2doeFFaeTQwX2s/view?usp=sharing))
some of these documents will then be filled with your start pack data, and require your signature (eg. project [Conflict of Interest form](https://drive.google.com/file/d/0B6h4FGwRepkSS05MOTVjZERkeU0/view?usp=sharing))

We'll need to force the user through viewing the essential ones, and marking that they have.

**Comment by JMurphyWeb** [comment]
Understood. 

@jonscottfilm is it enough to give the links to the user, and ask them to check a box saying they have read?

Also this suggests the PM should be able to indicate which of the file uploads must be read, and which are just FYI.

For now I think we don't need to worry too much about the documents that need to be filled in with start pack data, this functionality can be added once we start that feature on sprint 3.

**Comment by jonscottfilm** [comment]
The easiest solution for PM would be to make **all** files 'must be viewed' by the contractor. So, whatever the easiest method is to implement that...?
One link eg: 'Project Documents' goes to list of all files uploaded, can view all, or individually, but can only progress by checking 'viewed all' box, unless the system records the 'user open file'??

**Comment by JMurphyWeb** [comment]
That sounds like a good solution 😀

**Comment by JMurphyWeb** [comment]
@jonscottfilm  as agreed, removing from milestone sprint 2.

**Comment by jonscottfilm** [comment]
Strictly speaking should happen prior to accepting offer, yes. However, it doesn't happen now (on paper), so let's put low priority for now. They will see the documents (merged) seconds later having completed their startpack, and hitting accept....

**Comment by JMurphyWeb** [comment]
@jonscottfilm removing from Sprint 3
